src/ontology/cl-edit.owl

```diff
@@ -28983,6 +28984,19 @@ AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4030031 "2022-11-30T1
 AnnotationAssertion(rdfs:label obo:CL_4030031 "interstitial cell")
 EquivalentClasses(obo:CL_4030031 ObjectIntersectionOf(obo:CL_0000003 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0005169)))
 
+# Class: obo:CL_4030032 (valve interstitial cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref <https://orcid.org/0000-0001-7655-4833>) Annotation(oboInOwl:hasDbXref "PMID:25311230"^^xsd:string) Annotation(oboInOwl:hasDbXref "https://doi.org/10.1098/rsif.2017.0580"^^xsd:string) Annotation(oboInOwl:hasDbXref "https://doi.org/10.1161/ATVBAHA.120.314789"^^xsd:string) Annotation(oboInOwl:hasDbXref "https://doi.org/10.1161/JAHA.117.006339"^^xsd:string) Annotation(oboInOwl:hasDbXref "https://doi.org/10.2353/ajpath.2007.070251"^^xsd:string) obo:IAO_0000115 obo:CL_4030032 "A cardiac valve cell that resides in a valve leaflet. Along with valve endothelial cells, a valve interstitial cell maintains tissue homeostasis for the function of cardiac valves through secreting biochemical signals, matrix proteins and matrix remodeling enzymes."^^xsd:string)
```

Comment:
I would prefer the definiendum to be "An interstitial cell
that is part of the cardial valve" given the logical definition.